fsspec/asyn.py

```diff
@@ -760,11 +760,11 @@ async def _glob(self, path, maxdepth=None, **kwargs):
         detail = kwargs.pop("detail", False)
 
         if not has_magic(path):
-            if await self._exists(path):
+            if await self._exists(path, **kwargs):
```

Comment:
Can you please also fix these same lines in spec.py:AbstractFileSystem?

Reply:
They are fixed already: https://github.com/fsspec/filesystem_spec/blob/3c80a530c18f71518239b4215d4adcff69917a69/fsspec/spec.py#L577

Reply:
Sorry, I must have been looking at an old version...